### PR TITLE
Multiple dividends on the same day  should not be merged

### PR DESCRIPTION
**Pull request by jamsix**
FURS doesn't want multiple dividends from the same company paid on the same day being merged into one entry (https://github.com/jamsix/ib-edavki/issues/42).
This PR decouples such dividends, yet it still removes the dividends that have been reversed. The assumption is that a dividend reversal always has the same `dateTime` and the opposite `amount` of the original dividend.

### CODE REVIEW

File: ib_edavki.py [hunk 1]
```diff
@@ -1164,24 +1164,30 @@ def main():
...
-                mergedDividend["taxEUR"] = mergedDividend["taxEUR"] + dividend["taxEUR"]
-                merged = True
-                break
-        if merged == False:
-            mergedDividends.append(dividend)
-    dividends = mergedDividends
+    """ Dividends can be reversed. If there is a reversal, remove both the reversal
+        and the original dividend.
+        There is no unique identificator that binds a reversal to a dividend, the
+        assumption is that reversal ammount, date and securityID match the reversed
```
Comment (pronebird) [inline ib_edavki.py:1170]: nit: amount with single m

**Comment by pronebird** [comment]
Thanks for this PR. I haven't filed my reports yet, but I clearly remember Barrick doing some nasty reversals there in my account.

File: ib_edavki.py [hunk 2]
```diff
@@ -1164,24 +1164,30 @@ def main():
...
+                    and (
+                        dividend["securityID"] == reversal["securityID"]
+                        or dividend["symbol"] == reversal["symbol"]
+                    )
+                ):
+                    print(
+                        "%s %s dividend of %s has been reversed, removing."
+                        % (dividend["symbol"], dividend["dateTime"], dividend["amount"])
+                    )
+                    dividends.remove(dividend)
```
Comment (pronebird) [inline ib_edavki.py:1188]: Is it safe to mutate the collecting that's being iterated over?

Comment (jamsix) [reply ib_edavki.py:1188]: The loop is not over the `dividends` list, it's over it's sliced copy (`dividends[:]`). Is that what you had in mind? I should probably use `dividends.copy()` just to make it more readable.

Comment (pronebird) [reply ib_edavki.py:1188]: Yes. Makes sense. Thanks. I usually use `.copy()` but doesn't matter.

**Pull request merged**